resources/prefs.json

```diff
@@ -4,23 +4,32 @@
   "dom.canvas-text.enabled": true,
   "dom.compositionevent.enabled": false,
   "dom.customelements.enabled": true,
+  "dom.document.dblclick_dist": 1,
+  "dom.document.dblclick_timeout": 300,
   "dom.forcetouch.enabled": false,
+  "dom.fullscreen.test": false,
   "dom.gamepad.enabled": false,
+  "dom.microdata.enabled": true,
   "dom.microdata.testing.enabled": true,
   "dom.mouseevent.which.enabled": false,
   "dom.mutation_observer.enabled": true,
   "dom.offscreen_canvas.enabled": false,
   "dom.permissions.enabled": false,
   "dom.permissions.testing.allowed_in_nonsecure_contexts": false,
+  "dom.serviceworker.enabled": true,
   "dom.serviceworker.timeout_seconds": 60,
   "dom.servoparser.async_html_tokenizer.enabled": false,
+  "dom.svg.enabled": true,
```

Comment:
Sorry, we need to set the following preferences to false:
* dom.svg.enabled
* dom.serviceworker.enabled

Reply:
Thanks. I couldn't see anything about those in the logs. Where did you find them?

Reply:
That's based on this test output:
```
{
    "status": "FAIL", 
    "group": "default", 
    "message": "assert_true: If this is failing: DANGER, are you sure you want to expose the new interface ServiceWorkerContainer to all webpages as a property on the global? Do not make a change to this file without review from jdm or Ms2ger for that specific change! expected true got false", 
    "stack": "test_interfaces/<@http://web-platform.test:8000/_mozilla/mozilla/interfaces.js:70:7\nTest.prototype.step@http://web-platform.test:8000/resources/testharness.js:1587:20\ntest@http://web-platform.test:8000/resources/testharness.js:544:21\ntest_interfaces@http://web-platform.test:8000/_mozilla/mozilla/interfaces.js:2:3\n@http://web-platform.test:8000/_mozilla/mozilla/interfaces.html:13:1\n", 
    "subtest": "Interfaces exposed on the window", 
    "test": "/_mozilla/mozilla/interfaces.html", 
    "line": 241148, 
    "action": "test_result", 
    "expected": "PASS"
}
```

Reply:
@jdm do you know how I can find those results now? They aren't shown in the list of checks at the bottom of this issue any more, since there have been subsequent commits.

Reply:
Also, the errors are because the preferences are not found in the backing structure. They don't need to be present in `prefs.json` if they have a default value, and I thought it better to not include them there since they are only used in tests.